sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/README.md

```diff
@@ -0,0 +1,171 @@
+# Azure Key Vault Secret client library for Python
+Azure Key Vault is a cloud service that provides a secure storage of secrets, such as passwords and database connection strings. Secret client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge, backup, restore and and list the secrets and its versions.
+
+Use the secret client library to create and manage secrets.
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets) | [Package (Pypi)](TODO) | [API reference documentation](TODO) | [Product documentation](TODO)
+## Getting started
+### Install the package
+Install the Azure Key Vault client library for Python with pip:
+
+
+`$ pip install azure-security-keyvault
+`
+
+### Prerequisites
+* An [Azure subscription](https://azure.microsoft.com/free/).
+* Python 2.7, 3.4 or later.
+* An existing Key Vault. You can create this by following the instructions on [this article](https://docs.microsoft.com/en-gb/azure/key-vault/quick-create-portal).
+
+### Authenticate the client
+In order to interact with the Key Vault service for managing secrets, you'll need to create an instance of the SecretClient class. For this you will need a vault url, client secret credentials (client id, client secret, tenant id) and a [resource url](https://vault.azure.net).
+
+### Get credentials
+You can find credential information in [Azure Portal](https://portal.azure.com/).
```

Comment:
See comments in PR https://github.com/Azure/azure-sdk-for-java/pull/3644. The closest the documents are, the better